src/test/kotlin/org/stellar/walletsdk/utils/AccountReservedBalanceTest.kt

```diff
@@ -0,0 +1,28 @@
+package org.stellar.walletsdk.utils
+
+import kotlin.test.assertEquals
+import org.junit.jupiter.api.DisplayName
+import org.junit.jupiter.api.Test
+import org.stellar.sdk.responses.AccountResponse
+import org.stellar.walletsdk.helpers.objectFromJsonFile
+
+@DisplayName("accountReservedBalance")
+internal class AccountReservedBalanceTest {
+  @Test
+  fun `basic funded account`() {
+    val account =
+      objectFromJsonFile("src/test/resources/account_basic.json", AccountResponse::class.java)
+    val reservedBalance = accountReservedBalance(account)
+
+    assertEquals(1.0, reservedBalance.toDouble())
+  }
+
+  @Test
+  fun `more complex account`() {
+    val account =
+      objectFromJsonFile("src/test/resources/account_full.json", AccountResponse::class.java)
```

Comment:
The path `src/test/resources/account_full.json` is reliable because it depends on the running context of the JUnit test process. 

Here is a good example of [how to read a file from the resource folder](https://mkyong.com/java/java-read-a-file-from-resources-folder/).